vhdbuilder/parts/k8s/cloud-init/artifacts/docker-slice.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
```

Comment:
does this  mean we need the same for cse?

Reply:
You mean for non-vhd clusters?